targets/Program.cs

```diff
@@ -49,6 +53,16 @@ public static async Task Main(string[] args)
                 var source = Path.GetFullPath("./MinVer/bin/Release/");
                 var version = Path.GetFileNameWithoutExtension(Directory.EnumerateFiles(source, "*.nupkg").First()).Split("MinVer.", 2)[1];
 
+                File.WriteAllText(
+                    Path.Combine(testProject, "global.json"),
+@"{
+  ""sdk"": {
+    ""version"": ""3.1.300""
+  }
+}
+"
+                    );

```

Comment:
I'm late to the party, but here's a simpler way:
```
dotnet new globaljson --sdk-version 3.1.300 --force
```

Reply:
Thanks @thomaslevesque!

BTW why would I need the `--force` option?

Reply:
FYI — https://github.com/adamralph/minver/pull/350

Reply:
To overwrite the existing file, like your current code is doing